TestDevOps/Views/Shared/_Layout.cshtml

```diff
@@ -3,7 +3,7 @@
 <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>@ViewData["Title"] - TestDevOps</title>
+    <title>@ViewData["Title"] - DevOps</title>
```

Comment:
update the title to testing pull request

Reply:
updated. Please review the new changes. and merge to dev main